### PR TITLE
No file contents fix in "spacewalk-report config-files"

### PR DESCRIPTION
**Pull request by vzhestkov**
## What does this PR change?

Fixes the issue: https://github.com/uyuni-project/uyuni/issues/2484

## GUI diff

Before:
```
# spacewalk-report.orig config-files
org_id,channel_id,channel,channel_type,path,file_type,file_id,revision,is_binary,contents,delim_start,delim_end,username,groupname,filemode,symbolic_link,selinux_ctx
1,1,test-label,normal,/tmp/jom-test,file,2,1,N,<memory at 0x7f4c0d3531c8>,{|,|},root,root,644,,
1,1,test-label,normal,/tmp/test,file,1,1,N,<memory at 0x7f4c0d353288>,{|,|},root,root,644,,
1,1,test-label,normal,/tmp/test-bin,file,9,1,Y,<memory at 0x7f4c0d3531c8>,{|,|},root,root,644,,
1,2,test-state,state,/init.sls,sls,3,1,N,<memory at 0x7f4c0d353288>,{|,|},root,root,644,,
1,6,test-state-imp,state,/init.sls,sls,7,1,N,<memory at 0x7f4c0d3531c8>,{|,|},root,root,644,,
```

After:
```
# spacewalk-report config-files
org_id,channel_id,channel,channel_type,path,file_type,file_id,revision,is_binary,contents,delim_start,delim_end,username,groupname,filemode,symbolic_link,selinux_ctx
1,1,test-label,normal,/tmp/jom-test,file,2,1,N,"just one more test
еще один тест",{|,|},root,root,644,,
1,1,test-label,normal,/tmp/test,file,1,1,N,"just a test
просто тест
",{|,|},root,root,644,,
1,1,test-label,normal,/tmp/test-bin,file,9,1,Y,<binary data>,{|,|},root,root,644,,
1,2,test-state,state,/init.sls,sls,3,1,N,"test-file-mgmt:
  file.managed:
    - name: /tmp/test
    - content: |
      just a test
      one more",{|,|},root,root,644,,
1,6,test-state-imp,state,/init.sls,sls,7,1,N,,{|,|},root,root,644,,
```

## Documentation
- No documentation needed: restoring the original behavior

## Test coverage
- No tests: already covered

## Links

Fixes https://github.com/uyuni-project/uyuni/issues/2484

## Changelogs

- Fixes no file content in `spacewalk-report config-files`
- Write `<binary data>` placeholder instead of dumping binary data to the output

## Re-run a test

If you need to re-run a test, please mark the related checkbox, it will be unchecked automatically once it has re-run:

- [ ] Re-run test "changelog_test"
- [ ] Re-run test "backend_unittests_pgsql"
- [ ] Re-run test "ruby_rubocop"
- [ ] Re-run test "schema_migration_test_pgsql"
- [ ] Re-run test "susemanager_unittests"

### CODE REVIEW

File: reporting/spacewalk-reports.changes [hunk 1]
```diff
@@ -1,3 +1,9 @@
+-------------------------------------------------------------------
+Tue Oct 27 17:29:16 MSK 2020 - vzhestkov@suse.com
```
Comment (mcalmer) [inline reporting/spacewalk-reports.changes:2]: No headers please as this will be added by our tooling when we submit the changes to the buildservice.

**Review by mcalmer — CHANGES_REQUESTED** [review]

File: reporting/spacewalk-reports.changes (hunk 1 above)
Comment (vzhestkov) [reply reporting/spacewalk-reports.changes]: @mcalmer , fixed. Thanks

**Review by mcalmer — APPROVED** [review]

**Review by brejoc — APPROVED** [review]
lgtm!

**Comment by mcalmer** [comment]
@vzhestkov all approved but not merged yet, why?

**Pull request merged**